engine/gamesys/proto/gamesys/camera_ddf.proto

```diff
@@ -170,3 +170,56 @@ message ReleaseCameraFocus {}
  * end
  * ```
  */
+
+/*# [type:float] camera projection
+ *
+ * [mark:READ ONLY] The calcualted projection matrix of the camera.
+ * The type of the property is matrix4.
+ *
+ * @name projection
+ * @property
+ *
+ * @examples
+ *
+ * ```lua
+ * function init(self)
+ *   local projection = go.get("#camera", "projection")
+ * end
+ * ```
+ */
+
+/*# [type:float] camera view
+ *
+ * [mark:READ ONLY] The calcualted view matrix of the camera.
+ * The type of the property is matrix4.
+ *
+ * @name view
+ * @property
+ *
+ * @examples
+ *
+ * ```lua
+ * function init(self)
+ *   local view = go.get("#camera", "view")
+ * end
+ * ```
+ */
+
+/*# [type:float] camera aspect ratio
+ *
+ * The aspect ratio of the camera. Used when calculating the
+ * projection of a perspective camera.
+ * The type of the property is number.
```

Comment:
Perhaps better to describe what this ratio represents?
E.g. the relationship between the horizontal/vertical fov angle (or something)

Reply:
Added!